plugins/catalog-backend/src/catalog/types.ts

```diff
@@ -210,6 +217,15 @@ export interface EntitiesCatalog {
    * @param request - Request options
    */
   facets(request: EntityFacetsRequest): Promise<EntityFacetsResponse>;
+
+  /**
+   * Fetches unprocessed entities.
+   *
+   * @param request = request options
+   */
+  unprocessed(
+    request: UnprocessedEntitiesRequest,
+  ): Promise<HydratedRefreshState[]>;
```

Comment:
We should probably wrap this up into an object instead of returning just an array, much better for evolving the API over time without breaking the response type if we want to add another field or something.